src/music_kraken/utils/shared.py

```diff
@@ -2,8 +2,11 @@
 import logging
 import tempfile
 import os
+import io
+import configparser
+from sys import platform as current_os
 
-
+USER_XDG_DIR_FILE = os.path.expanduser("~/.config/user-dirs.dirs")
```

Comment:
Just a personal preference, but for the sake of clarity, I would prefer `XDG_USER_DIRS_FILE` for the name of that variable.

Reply:
Even better, add a comment pointing to the freedesktop page (https://freedesktop.org/wiki/Software/xdg-user-dirs/).

Reply:
Hey thank you so much!
If you don't mind, could you please tell me how to link to lines of the changed files. I am really new to Pull Requests.

You're suggested changes look really good and should really be implemented. After @DistantThunder changed those things, I am gonna merge the pull request and work on refactoring the stuff you said. 

- Using Pathlib
- Moving most executing code to another module
- move `USER_XDG_DIR_FILE` out of the scope

Note: `shared.py` is not meant to be a config file. I am implementing a config file later.

Reply:
Sure, no problem.

When you hover a line on files changed tab ([here for this PR](https://github.com/HeIIow2/music-downloader/pull/6/files)), you can see a blue `+` sign appearing. If you click on it, you can add a comment for that line. If you want to make a comment about a piece of code taking more than one line, you can drag and drop the plus vertically.

In github, if you are not sure about your comments, or if you want a bunch of comments to be sent together, you can click on the "start a review button", instead of "add a single comment". Everything you write from there is a draft. Then you have to click on the green button "review changes" on the top right, to finalise your review, and send it with an optional comment. From there, if you click approve, github considers that despite your comments, you are okay with the pull request being merged as it is. If you click on "request changes", that means to github that your comments should be addressed before you can approve this PR. And if you click on comment, you are neutral.

There are other convenient features to pull request review. I recommend you have a look that the github documentation when you have the time:

- [About pull request reviews](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews)
- [Reviewing proposed changes in a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) (I especially like suggestion blocks, part 7)
- [Viewing a pull request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/viewing-a-pull-request-review)

Of course you can also use APIs to review code. This is a bit more tedious without proper tooling tough:

- using the GrapQL API: [addPullRequestReview](https://docs.github.com/en/graphql/reference/mutations#addpullrequestreview), [addPullRequestReviewComment](https://docs.github.com/en/graphql/reference/mutations#addpullrequestreview), [addPullRequestReviewThread](https://docs.github.com/en/graphql/reference/mutations#addpullrequestreviewthread)
- using the REST API: [Pull request reviews](https://docs.github.com/en/rest/pulls/reviews#about-the-pull-request-reviews-api)

Another thing you can do is prevent pull request from being merged if they have not been reviewed and approved: [About protected branches](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches)

Reply:
Hi thanks. This really helped. One last thing. Can I do the `request changes` thingy for you're comments? They are really on point.